source/_posts/2021-07-07-release-20217.markdown

```diff
@@ -80,10 +79,9 @@ Genie you can now change the room size for your diffuser.
 ## Trigger conditions and trigger IDs
 
 If you are creating some complex automations in YAML, you might be familiar with
-this. Consider an big automation, with a whole bunch of triggers. But how
-would you know which of those triggers actually triggered the automation?
+this. Consider a big automation, with a whole bunch of triggers. But how would you know which of those triggers actually triggered the automation?
```

Comment:
I don't see a change in this line, besides removing a new line that has no effect?

Reply:
Yeah sorry, I think I made changes and reverted them again in some blocks, that's why I changed the new lines.

I'll revert that.